src/frontend/src/flows/recovery/displaySeedPhrase.ts

```diff
@@ -9,14 +9,19 @@ import { toast } from "../../components/toast";
 import copyJson from "./displaySeedPhrase.json";
 
 const displaySeedPhraseTemplate = ({
+  op,
   seedPhrase,
   onContinue,
+  cancel,
   copyPhrase: copyPhrase_,
   i18n,
 }: {
+  /* whether the phrase is created for the first time or just reset */
+  op: "create" | "reset";
```

Comment:
`op` as `operation`?

Reply:
yeah, do you think `operation` is clearer? any other suggestions? I struggled a bit with this one